docs/wfc3tools/pstack.rst

```diff
@@ -57,36 +32,73 @@ The plotting data is returned as two arrays, to save them into variables issue t
 Parameters
 ==========
 
-* input [file]
+* filename : str
     Input MultiAccum image name.  This should be either  a  _ima  or _raw  file, containing all the data from multiple readouts.  You must specify just the file name, with no extension designation.
 
-* col [integer]
+* column : int, default=0
     The column index of the pixel to be plotted.
 
-* row [integer]
+* row : int, default=0
     The row index of the pixel to be plotted.
 
-* extname = "sci" [string, allowed values: sci | err | dq | samp | time]
-   Extension name (EXTNAME keyword value) of data to plot.
+* extname : str, default="sci"
+   Extension name (EXTNAME keyword value) of data to plot. Allowed values are "sci", "err", "dq", "samp", and "time".
 
-* units = "counts" [string, allowed values: counts | rate]
-   Plot "sci" or  "err"  data  in  units  of  counts  or  countrate ("rate").   Input data can be in either unit; conversion will be performed automatically.  Ignored when  plotting  "dq",  "samp", or "time" data.
+* units : str, default="counts"
+   Plot "sci" or  "err"  data  in  units  of  counts  or  countrate ("rate").   Input data can be in either unit; conversion will be performed automatically.  Ignored when  plotting  "dq",  "samp", or "time" data. Allowed values are "counts" and "rate".
 
-* title = "" [string]
+* title : str, default=None
    Title  for  the  plot.   If  left  blank,  the name of the input image, appended with  the  extname  and  column  and  row  being plotted, is used.
 
-* xlabel = "" [string]
+* xlabel : str, default=None
    Label  for  the  X-axis  of the plot.  If left blank, a suitable default is generated.
 
-* ylabel = "" [string]
+* ylabel : str, default=None
    Label for the Y-axis of the plot.  If  left  blank,  a  suitable default  based  on the plot units and the extname of the data is generated.
 
-* plot = True [bool]  set plot to false if you only want the data returned
+* plot : bool, default=True
+    If False, return data and do not plot.
+
+Returns
+=======
+
+* xaxis : numpy.ndarray
+   Array of x-axis values that will be plotted.
```

Comment:
Data is only returned if `plot=False` so why do the returns say "that will be plotted".  Seems confusing.

Reply:
i've never used this function, lets address this later once it's tested